build-system/tasks/build.js

```diff
@@ -81,24 +87,42 @@ function polyfillsForTests() {
   return compileJs('./src/', 'polyfills.js', './build/');
 }
 
+/**
+ * The default task run when `gulp` is executed
+ */
+const defaultTask = gulp.series(watch, serve);
+
+module.exports = {
+  build,
+  defaultTask,
+  watch,
+};
+
 /* eslint "google-camelcase/google-camelcase": 0 */
 
-gulp.task('build', 'Builds the AMP library', maybeUpdatePackages, build, {
-  options: {
-    config: '  Sets the runtime\'s AMP_CONFIG to one of "prod" or "canary"',
-    extensions: '  Builds only the listed extensions.',
-    extensions_from: '  Builds only the extensions from the listed AMP(s).',
-    noextensions: '  Builds with no extensions.',
-  },
-});
-gulp.task('watch', 'Watches for changes in files, re-builds when detected',
-    maybeUpdatePackages, watch, {
-      options: {
-        with_inabox: '  Also watch and build the amp-inabox.js binary.',
-        with_shadow: '  Also watch and build the amp-shadow.js binary.',
-        extensions: '  Watches and builds only the listed extensions.',
-        extensions_from: '  Watches and builds only the extensions from the ' +
-            'listed AMP(s).',
-        noextensions: '  Watches and builds with no extensions.',
-      },
-    });
+build.description = 'Builds the AMP library';
```

Comment:
I'm curious about this pattern. Is this a gulp v4 thing?

Would it make more sense to modify the objects before exporting?

Reply:
Yep, it's a gulp v4 thing.

The reason I kept the options last is because `/* eslint "google-camelcase/google-camelcase": 0 */` disables the check for the rest of the file, so I only wanted the options to be exempted from the check.